habit/src/components/TextInput.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { TextInput } from 'react-native'
+import PropTypes from 'prop-types';
+
+export default function CustomTextInput(props){
+    return  <TextInput 
+                { ...props }
+                style={ props.style }
+            />
+}
+
+// CustomTextInput.propTypes = {
```

Comment:
Let's remove this, if not needed

Reply:
not now but the component is going to grow and we'll need it